src/components/RecipeModal/RecipeModal.jsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react"
+import Modal from "react-bootstrap/Modal"
+import { Container, Row, Col, Button } from "react-bootstrap" 
+import "./RecipeModal.css"
+
+export default function RecipeModal(props) {
+    if (!props.show) {
+        return null
+    }
+    return (
+      <Modal {...props} scrollable={true} size="lg" aria-labelledby="contained-modal-title-vcenter" centered>
+        <Modal.Header closeButton>
+          <Modal.Title id="contained-modal-title-vcenter">
+            {props.title}
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body className="show-grid">
+          <Container>
+            {/* first row - recipe overview */}
+            <Row className="modal-overview">
+                <Col>
+                Serving Time: {props.recipeinfo.readyInMinutes} mins
+                </Col>
+                <Col>
+                Servings: {props.recipeinfo.servings}
+                </Col>
+            </Row>
+            {/* recipe content */}
+            <Row>
+                {/* ingredient content */}
+                <Col>
+                    <p className="modal-headings">Ingredients:</p>
+                    {
+                        props.recipeinfo.extendedIngredients.map((ingredient) => (
+                            <p key={ingredient.id}>* {ingredient.original}</p>
+                        ))
+                    }
+                </Col>
+                {/* instructions content */}
+                <Col>
+                    <p className="modal-headings">Steps:</p>
+                    {
+                        props.instructions[0].steps.map((curStep) => (
```

Comment:
Why are you mapping instructions[0]? Are you not just passing in the instructions you're displaying?

Reply:
The API returns an array of Objects that hold instructions, and it's the main instruction at index 0, and then supplementary instructions. For example for this one, it was a recipe for pancakes but it also had another set of instructions to create the butter for the pancakes. So I thought only getting the first Object would be necessary

Reply:
Ah ok got it! Can you add a comment in there explaining that's what you're doing?